project/src/views/new-evaluation/technicalLevelPicker.js

```diff
@@ -0,0 +1,39 @@
+const TechnicalLevelPickerHeader= function(options = {}) {
+    const headers = options.map(function(h) {        
+      return `<th class="tbl-head tbl-head-skin">${h}</th>`;
+    });
+    const concatHeaders = headers.join("");
+    return concatHeaders;
+}
+
+const TechnicalLevelPickerColumn = function(options = {}) {
+    const allColumns = options.map((cd) => {
+        const columns = [`<td>`];
+        const columns2 = cd.inputLevels.map((ip) => `<input type="radio" id= "${ip}" name="level">` )
```

Comment:
Why not `columns34543`? :smile:

Reply:
renamed it :d also returned without declaring a needless constant